t3f/ops_test.py

```diff
@@ -693,6 +707,20 @@ def testGatherNDBatch(self):
       self.assertAllClose(res_np_v, des_v)
       self.assertAllClose(res_pl_v, res_pl_v)
 
+  def testCoreRenormBatch(self):
+      a = initializers.random_tensor_batch(3 * (10,), tt_rank=7, batch_size=5)
```

Comment:
This class is for testing matrices, not tensors

Reply:
But this function was in `TTTensorBatchTest` class ...